packages/datadog-plugin-graphql/src/index.js

```diff
@@ -425,7 +424,13 @@ function getVariablesFilter (config) {
   if (typeof config.variables === 'function') {
     return config.variables
   } else if (config.variables instanceof Array) {
-    return variables => pick(variables, config.variables)
+    return variables => {
+      const newVariables = {}
+      for (const key in config.variables) {
+        newVariables[key] = config.variables[key]
```

Comment:
```suggestion
        newVariables[key] = variables[key]
```

Reply:
If this is not failing a test then a test should be added.